lib/github_changelog_generator/generator/entry.rb

```diff
@@ -26,41 +26,38 @@ def initialize(options = Options.new({}))
     # @param [String] newer_tag_link Name of the newer tag. Could be "HEAD" for `Unreleased` section.
     # @param [Time] newer_tag_time Time of the newer tag
     # @param [Hash, nil] older_tag Older tag, used for the links. Could be nil for last tag.
-    # @return [String] Ready and parsed section
-    def create_entry_for_tag(pull_requests, issues, newer_tag_name, newer_tag_link, newer_tag_time, older_tag_name) # rubocop:disable Metrics/ParameterLists
+    # @return [String] Ready and parsed section content.
+    def generate_entry_for_tag(pull_requests, issues, newer_tag_name, newer_tag_link, newer_tag_time, older_tag_name) # rubocop:disable Metrics/ParameterLists
       github_site = @options[:github_site] || "https://github.com"
       project_url = "#{github_site}/#{@options[:user]}/#{@options[:project]}"
 
-      set_sections_and_maps
+      create_sections
```

Comment:
is it worth changing this method name to `generate_sections`? it would match better with everything around it.

Reply:
Reading the code, my personal take is “create_” reads easier. 

We could move to that naming instead? But to me, that’s not a merge blocker for this change. 

Thanks for keeping sharp eyes on This developing PR, @eputnam!

Reply:
This method is about filling the `@sections` array with the `Section.new` objects to be filled out later on. I tried to make any `generate_*` methods be ones that return generated string content for the changelog.